com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -371,11 +371,20 @@ private static List<File> downloadExes(Platform platform, String variant, String
                 logger.info("Download: %s", url);
                 File file = new File(downloadFolder, exeName);
                 HttpUtil http = new HttpUtil();
-                http.downloadToFile(url, file);
+
+                // Try to download the first URL
+                try {
+                    http.downloadToFile(url, file);
+                } catch (Exception downloadException) {
+                    // If the first URL fails, try the fallback URL for 'win32' platform
+                    URL fallbackUrl = new URL(String.format(artifactsURL + "%s/engine/%s/%s", EngineVersion.sha1, platform.getOs(), exeName));
+                    logger.info("First download attempt failed, trying fallback URL: %s", fallbackUrl);
+                    http.downloadToFile(fallbackUrl, file);
+                }
```

Comment:
`x86_win32` for some reason uses just `win32` in its instead of full platform name. So we need to try this one as well